### PR TITLE
feat(#minor); BadgerDAO Subgraph; Fix deposit amount bug

### PR DESCRIPTION
**Pull request by harsh9200**
Fixes: #2163 
Deployment: https://thegraph.com/hosted-service/subgraph/messari/badgerdao-ethereum?version=pending

### CODE REVIEW

**Comment by dmelotik** [comment]
Can you increment versions (minor version change)

File: subgraphs/badgerdao/src/modules/Deposit.ts [hunk 1]
```diff
@@ -92,7 +92,13 @@ export function Deposit(
       constants.BIGINT_ZERO
     );
 
-    depositAmount = totalSupply.minus(vault.outputTokenSupply!);
+    const sharesMinted = totalSupply.minus(vault.outputTokenSupply!);
+
+    depositAmount = vault.outputTokenSupply!.isZero()
```
Comment (dmelotik) [inline subgraphs/badgerdao/src/modules/Deposit.ts:97]: Shouldn't this be done after we set the new vault.inputTokenBalance and vault.outpuTokenSupply?

Comment (harsh9200) [reply subgraphs/badgerdao/src/modules/Deposit.ts:97]: No, this needs to be done with the old values as we need to find the amount deposited in the pool using the shares minted during a deposit transaction.

File: subgraphs/balancer-forks/src/prices/oracles/InchOracle.ts
Comment (dmelotik) [inline subgraphs/balancer-forks/src/prices/oracles/InchOracle.ts:1]: Out of scope of this PR?

**Review by dmelotik — APPROVED** [review]
Approving and can be merged once that last comment is fixed! Nice work @harsh9200

File: subgraphs/balancer-forks/src/prices/oracles/InchOracle.ts
Comment (harsh9200) [reply subgraphs/balancer-forks/src/prices/oracles/InchOracle.ts:1]: No, this is a required change for the fix, cause this bug was related to the digg token pricing and only the inchOracle method returns the correct price for that token.

**Pull request merged**